src/main/java/br/com/mercadolivre/loja/user/User.java

```diff
@@ -15,6 +17,8 @@
     private String login;
     private String password;
 
+
+    @Deprecated
     public User() {}
```

Comment:
Os construtores podem ser com modificador default.

```suggestion
    User() {}
```